.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,89 @@
+# Description: Run this by visiting https://github.com/brave/brave-core/actions/workflows/bump-deps.yaml
+# and clicking the `Run workflow` button on the top-right.
+# You can also run it from cli by: `gh workflow run bump-deps.yaml [-f dep=vendor/web-discovery-project] [-f ref=commit-hash]`
+# If `ref` is not given, we will try to use the latest commmit (tip) of a branch called `master` or `main`
+
+name: Bump/update dependency [ref] in DEPS
```

Comment:
Sorry, I should have said it's not literally. `Update dependency in DEPS`

Reply:
File should be named `update-dep.yml`

Reply:
Done